src/debug/daccess/CMakeLists.txt

```diff
@@ -63,4 +63,4 @@ if(CLR_CMAKE_PLATFORM_UNIX)
   )
 
   add_dependencies(daccess coreclr dactablerva_header)
-endif(CLR_CMAKE_PLATFORM_UNIX)
+endif(CLR_CMAKE_PLATFORM_UNIX AND NOT CLR_CROSS_COMPONENTS_BUILD)
```

Comment:
Instead of these changes could you change this line from 
add_dependencies(daccess coreclr dactablerva_header)  
to
add_dependencies(daccess dactablerva_header)

and see if it works.